addons/mail_bot/static/src/owl/components/notification_request/notification_request.js

```diff
@@ -0,0 +1,72 @@
+odoo.define('mail_bot.component.NotificationRequest', function (require) {
+'use strict';
+
+const PartnerImStatusIcon = require('mail.component.PartnerImStatusIcon');
+const useStore = require('mail.hooks.useStore');
+
+var { _t } = require('web.core');
+
+const { Component } = owl;
+const { useDispatch } = owl.hooks;
+
+class NotificationRequest extends Component {
+
+    /**
+     * @override
+     * @param {...any} args
+     */
+    constructor(...args) {
+        super(...args);
+        this.storeDispatch = useDispatch();
+        this.storeProps = useStore((state, props) => {
+            return {
+                isMobile: state.isMobile,
+                partnerRootLocalId: state.partnerRootLocalId,
+            };
+        });
+    }
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * Handle the response of the user when prompted whether push notifications
+     * are granted or denied.
+     *
+     * @private
+     * @param {string} value
+     */
+    _handleResponseNotificationPermission(value) {
```

Comment:
not technically a handler, so move it in `// Private`

Reply:
ok